client/remoteclient/grpc_client.go

```diff
@@ -2,50 +2,58 @@ package remoteclient
 
 import (
 	"context"
-	"fmt"
 
-	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/proto"
+	"google.golang.org/grpc"
 
-	api "github.com/ligato/vpp-agent/api/genericmanager"
+	"github.com/ligato/vpp-agent/api/generic"
 	"github.com/ligato/vpp-agent/client"
 	"github.com/ligato/vpp-agent/pkg/models"
 	"github.com/ligato/vpp-agent/pkg/util"
 )
 
 type grpcClient struct {
-	remote api.GenericManagerClient
+	manager generic.ManagerClient
+	meta    generic.MetaServiceClient
 }
 
 // NewClientGRPC returns new instance that uses given service client for requests.
-func NewClientGRPC(client api.GenericManagerClient) client.ConfigClient {
-	return &grpcClient{client}
+func NewClientGRPC(conn *grpc.ClientConn) client.ConfigClient {
+	manager := generic.NewManagerClient(conn)
+	meta := generic.NewMetaServiceClient(conn)
+	return &grpcClient{
+		manager: manager,
+		meta:    meta,
+	}
 }
 
-func (c *grpcClient) KnownModels() ([]api.ModelInfo, error) {
+func (c *grpcClient) KnownModels(class string) ([]*client.ModelInfo, error) {
 	ctx := context.Background()
 
-	resp, err := c.remote.Capabilities(ctx, &api.CapabilitiesRequest{})
+	resp, err := c.meta.KnownModels(ctx, &generic.KnownModelsRequest{
+		Class: class,
+	})
 	if err != nil {
 		return nil, err
 	}
 
-	var modules []api.ModelInfo
+	var modules []*generic.ModelDescriptor
```

Comment:
Shouldn't this be models? The module is a part of the model spec... let's keep it consistent.